src/main/controllers/VideoHearingsController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from 'express';
+
+export default class VideoHearingController {
+
+  public get(req: Request, res: Response): void {
+    res.render('video-hearings', {
+      ...(req.t('video-hearings', { returnObjects: true })),
+    });
+  }
+
+  public post(req: Request, res: Response): void {    
+    if (req.body['video-hearing'] === 'yes' &&  req.body.saveButton === 'saveContinue') {
+      res.redirect('/steps-to-making-your-claim');
+    }
+    else if (req.body['video-hearing'] === 'no' && req.body.saveButton === 'saveContinue') {
+      res.redirect('/steps-to-making-your-claim');
+    }
+    else if (req.body['video-hearing'] === 'yes' && req.body.saveButton === 'saveForLater') { 
+      res.redirect('/your-claim-has-been-saved');
+    }
+    else if (req.body['video-hearing'] === 'no' && req.body.saveButton === 'saveForLater') { 
+      res.redirect('/your-claim-has-been-saved');
+    }
```

Comment:
This logic seems really overly convoluted. Could you please extract this check into a function and combine 1-2 `if` statements and `3-4` together?